upstream-community-operators/quay/quay.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+- currentCSV: quay.v1.0.2
```

Comment:
This package manifest doesn't look correct to me. It should be:
```
channels:
- name: stable
   currentCSV: quay.v1.0.2
defaultChannel: stable
packageName: quay
```

Reply:
@dinhxuanvu appears to be no difference. the order in the fields in the list is not an issue in YAML formatted files